pkg/bpf/bpffs.go

```diff
@@ -20,8 +20,11 @@ import (
 	"path"
 	"sync"
 
+	"bytes"
```

Comment:
Imports should be on 3 blocks, 1st for golang libs, 2nd for cilium libs, 3rd for 3rd party libraries.

Reply:
sure

Reply:
Done